lib/src/main/java/com/otaliastudios/transcoder/internal/thumbnails/DefaultThumbnailsEngine.kt

```diff
@@ -156,6 +173,7 @@ class DefaultThumbnailsEngine(
         if (stub != null) {
             log.i("removePosition Match: $positionUs :$stubs")
             stubs.remove(stub)
+            shouldSeek = true
```

Comment:
why?

Reply:
if a position has been removed from the stubs, next seeker call would come to the next in list timestamp, for which shouldSeek would've been false, since videoSnapshots wouldn't be able to mark it true now. hence forcing it

Reply:
Yeah.. it seems that it’s better (more optimized) to do it in removePosition.
Coz it’ll flush the codecs and we’ll provide the next frame to VideoSnapshots sooner